serokell-style.md

```diff
@@ -233,47 +230,47 @@ align by max width among them.
 
 Use singular when naming modules (e.g. use `Data.Map` and
 `Data.ByteString.Internal` instead of `Data.Maps` and
-`Data.ByteString.Internals`). Sometimes it's acceptable to use plural
+`Data.ByteString.Internals`). Sometimes it is acceptable to use plural
 (e. g. `Types`, `Instances`).
 
 ### Export Lists
 
+All modules _must_ have explicit exports.
+
 Format export lists as follows:
```

Comment:
I would permit writing export list on one line if it's empty (like we do in Ariadne).